packages/@aws-cdk/aws-s3-deployment/test/lambda/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM public.ecr.aws/lambda/python:latest
+
+# add everything to /opt/awscli (this is where `aws` is executed from)
+ADD . /opt/awscli
```

Comment:
Maybe convert this to a `--volume` in a `docker run` command? Makes debugging easier

Reply:
How come? You can just `docker run -it XXX` and go into the container which contains everything. It's a pretty useful method.